packages/notifications/src/elements/Well.tsx

```diff
@@ -10,7 +10,9 @@ import PropTypes from 'prop-types';
 import { StyledWell } from '../styled';
 
 interface IWellProps {
+  /** Applies a background color to differentiate the well from the rest of the page */
   isRecessed?: boolean;
+  /** Applies a drop shadow to lift the well off of the page */
```

Comment:
I'm totally digging this. But tagging @allisonacs and @ctancinco for a consistency head-check. How design-explicit do we want to be here? Is the simpler "Applies a drop shadow" better?

Reply:
short and sweet is my vote. your edit works for me.